build failure

building with latest wlroots from Arch linux AUR fails. Here is the log:
FAILED: waybox/waybox@@waybox@exe/output.c.o 
cc -Iwaybox/waybox@@waybox@exe -Iwaybox -I../waybox -I../include -I/usr/include/libdrm -I/usr/include/pixman-1 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c11 -O0 -g -Wno-unused-parameter -DWL_HIDE_DEPRECATED  -MD -MQ 'waybox/waybox@@waybox@exe/output.c.o' -MF 'waybox/waybox@@waybox@exe/output.c.o.d' -o 'waybox/waybox@@waybox@exe/output.c.o' -c ../waybox/output.c
../waybox/output.c: In function ‘output_frame_notify’:
../waybox/output.c:27:29: error: invalid type argument of ‘->’ (have ‘struct wlr_surface_state’)
    .width = surface->current->width,
                             ^~
../waybox/output.c:28:30: error: invalid type argument of ‘->’ (have ‘struct wlr_surface_state’)
    .height = surface->current->height
                              ^~
../waybox/output.c:31:69: error: invalid type argument of ‘->’ (have ‘struct wlr_surface_state’)
         wlr_matrix_project_box(matrix, &render_box, surface->current->transform,
                                                                     ^~
ninja: build stopped: subcommand failed.

--- a/waybox/output.c
+++ b/waybox/output.c
@@ -24,11 +24,11 @@ void output_frame_notify(struct wl_listener *listener, void *data) {
 		}
 		struct wlr_box render_box = {
 			.x = 20, .y = 20,
-			.width = surface->current->width,
-			.height = surface->current->height
+			.width = surface->current.width,
+			.height = surface->current.height
 		};
 		float matrix[16];
-        wlr_matrix_project_box(matrix, &render_box, surface->current->transform,
+        wlr_matrix_project_box(matrix, &render_box, surface->current.transform,
 			    0, wlr_output->transform_matrix);
 		struct wlr_texture *texture = wlr_surface_get_texture(surface);
 	    wlr_render_texture_with_matrix(renderer, texture, matrix, 1.0f);

